mock/ext-api-dyson/get/binance-rpc-txs.js

```diff
@@ -0,0 +1,69 @@
+/// Binance chain block explorer RPC Mock, tx
+/// Returns:
+/// - Multi-transaction transaction for a specific address
+///   1. newman calls: http://localhost:8437/v1/binance/bnb1jeu6gscugy6l2wyatxthkh2hmer4hzevgcmf0q
+///   2. Block atlas call internally : https://{binance_rpc}/api/v1/txs?address=bnb1jeu6gscugy6l2wyatxthkh2hmer4hzevgcmf0q&page=1&rows=25&txType=TRANSFER
+///   3. Dyson response mock data: http://localhost:3347/binance-dex/api/v1/txs?address=bnb1jeu6gscugy6l2wyatxthkh2hmer4hzevgcmf0q&page=1&rows=25&txType=TRANSFER
+/// - empty response for other txHash'es
+
+// Example contains each type, BNB transfer, BEP2 transfer, BEP2 transfer as multisend transaction
+module.exports = {
+    path: '/binance-dex/v1/transactions',
```

Comment:
The path here does not match the paths used to DEX, shown in lines 2. and 3. above.  I think the correct would be '/binance-dex/v1/txs'.